otter/convergence/composition.py

```diff
@@ -66,3 +46,36 @@ def tenant_is_enabled(tenant_id, get_config_value):
     """
     enabled_tenant_ids = get_config_value("convergence-tenants")
     return (tenant_id in enabled_tenant_ids)
+
+
+def json_to_LBConfigs(lbs_json):
```

Comment:
I moved it out of gathering.py because it's not related to fetching data -- it's only to convert otter's own format for LB configs to the LBConfig object (actually, maybe it would make more sense in model.py?)